src/app/tests/suites/certification/Test_TC_LVL_2_1.yaml

```diff
@@ -23,116 +23,140 @@ tests:
       cluster: "DelayCommands"
       command: "WaitForCommissionee"
 
-    - label: "reads current Level attribute from DUT"
+    #Disabled due to issue - #11670
+    - label: "Reads the CurrentLevel attribute"
+      disabled: true
       command: "readAttribute"
-      attribute: "current Level"
+      attribute: "current level"
       response:
-          value: 254
+          value: null
```

Comment:
Why would it be null?  The actual value you would get here may be device-dependent; for a typical light on first bootup I would expect 254 (full brightness).

Note that the spec's default value for an attribute may not actually be the initial value on bootup; those are not the same thing!

Reply:
i will update with def value (254)

Reply:
The actual value after power-on here is DUT specific.